### ISSUE TITLE
Over 100%

### ISSUE THREAD

**Issue by rmazurk** [tjluoma/textbar-timemachine#3]
After downloading and running in the textbar, it shows me over 100% - e.g. currently 121%

**Comment by tjluoma** [comment]
Yup. Sometimes the numbers don't make sense. I'm just reporting what tmutil shows.

**Comment by rmazurk** [comment]
I have a strange feeling that the data is shown the other way around :) I have more Copied than Total data :) so it should be? Can you verify it with yourself?
